tox.ini

```diff
@@ -1,111 +1,113 @@
 [tox]
-envlist = py{27,33,34,35,36,37}-{cdh,hdp,core,contrib,apache,aws,gcloud,postgres,unixsocket, azureblob}, visualiser, pypy-scheduler, docs, flake8
+envlist = py{27,33,34,35,36,37}-{cdh,hdp,core,contrib,apache,aws,gcloud,postgres,unixsocket,azureblob,dropbox}, visualiser, pypy-scheduler, docs, flake8
 skipsdist = True
 
 [testenv]
 usedevelop = True
 install_command = pip install {opts} {packages}
-deps=
-  mock<2.0
-  moto==1.3.6
-  HTTPretty==0.8.10
-  nose<2.0
-  docker>=2.1.0
-  unittest2<2.0
-  boto<3.0
-  boto3>=1.4.4
-  s3transfer==0.1.13
-  sqlalchemy<2.0
-  elasticsearch<2.0.0
-  psutil<4.0
-  enum34>1.1.0
-  cdh,hdp: snakebite>=2.5.2,<2.6.0
-  cdh,hdp: hdfs>=2.0.4,<3.0.0
-  postgres: psycopg2<3.0
-  mysql-connector-python>=8.0.12
-  gcloud: google-api-python-client>=1.4.0,<2.0
-  py27-gcloud: avro
-  py33-gcloud,py34-gcloud,py35-gcloud,py36-gcloud: avro-python3
-  gcloud: google-auth==1.4.1
-  gcloud: google-auth-httplib2==0.0.3
-  google-compute-engine
-  coverage>=4.1,<4.2
-  codecov>=1.4.0
-  requests>=2.20.0,<3.0
-  unixsocket: requests-unixsocket<1.0
-  pygments
-  hypothesis[datetime]<4.0.0
-  selenium==3.0.2
-  pymongo==3.4.0
-  toml<2.0.0
-  responses<1.0.0
-  azure-storage
-  datadog==0.22.0
-  prometheus-client==0.5.0
+deps =
+    mock<2.0
+    moto==1.3.6
+    HTTPretty==0.8.10
+    nose<2.0
+    docker>=2.1.0
+    unittest2<2.0
+    boto<3.0
+    boto3>=1.4.4
+    s3transfer==0.1.13
+    sqlalchemy<2.0
+    elasticsearch<2.0.0
+    psutil<4.0
+    enum34>1.1.0
+    cdh,hdp: snakebite>=2.5.2,<2.6.0
+    cdh,hdp: hdfs>=2.0.4,<3.0.0
+    postgres: psycopg2<3.0
+    mysql-connector-python>=8.0.12
+    gcloud: google-api-python-client>=1.4.0,<2.0
+    py27-gcloud: avro
+    py33-gcloud,py34-gcloud,py35-gcloud,py36-gcloud: avro-python3
+    gcloud: google-auth==1.4.1
+    gcloud: google-auth-httplib2==0.0.3
+    google-compute-engine
+    coverage>=4.1,<4.2
+    codecov>=1.4.0
+    requests>=2.20.0,<3.0
+    unixsocket: requests-unixsocket<1.0
+    pygments
+    hypothesis[datetime]<4.0.0
+    selenium==3.0.2
+    pymongo==3.4.0
+    toml<2.0.0
+    responses<1.0.0
+    azure-storage
+    datadog==0.22.0
+    prometheus-client==0.5.0
+    dropbox: dropbox
```

Comment:
should we specify a version?

Reply:
Reading in PyPi, I have seen that dropbox 7.2.1 (March 2017) is the first release to mention support for Dropbox API v2 (which is the ones I have used).

Any release of dropbox >7.2.1 should work (currently they are at 9.3.0).
Should I specify a version?

Reply:
I'd be more concerned about setting an upper end. If dropbox follows semantic versioning (which luigi does not), we could set the version to something like `dropbox>=7.2.1,<7.3`.

Reply:
I have seen that dropbox somewhat follows semantic versioning (e.g they dropped support for v1 API in 8.0.0). 

Therefore, I am going to set the version to `dropbox>=9.3,<10.0` (since we know that the current version -9.3.0- is working OK, but I have not tested previous versions; and we cannot be sure that there won't be breaking changes in version 10 )

Reply:
I'm fine with that

Reply:
I have just added `dropbox>=9.3,<10.0` to tox.ini